packages/m/msgpack-cxx/package.yml

```diff
@@ -0,0 +1,20 @@
+name       : msgpack-cxx
+version    : 6.1.0
+release    : 1
+source     :
+    - https://github.com/msgpack/msgpack-c/releases/download/cpp-6.1.0/msgpack-cxx-6.1.0.tar.gz : 23ede7e93c8efee343ad8c6514c28f3708207e5106af3b3e4969b3a9ed7039e7
+homepage   : https://github.com/msgpack/msgpack-c/tree/cpp_master
+license    : BSL-1.0
+component  : programming.library
+summary    : msgpack for C++
+description: |
+    MessagePack is an efficient binary serialization format, which lets you exchange data among multiple languages like JSON, except that it's faster and smaller. Small integers are encoded into a single byte and short strings require only one extra byte in addition to the strings themselves.
+patterns   : /*
+builddeps  :
+    - libboost-devel
+setup      : |
+    %cmake_ninja
+build      : |
+    %ninja_build
+install    : |
+    %ninja_install
```

Comment:
We have a package for `msgpack-c` already; I wonder if it would be better to build both the C and C++ versions in the same package and pattern them out.

Reply:
Unfortunately, I don't think you can. I don't know why upstream does this, but if I understand it correctly the C and C++ versions are two completely different branches developed in parallel.